### ISSUE TITLE
Not able to interact with the Hammer.js Press event 

### ISSUE THREAD

**Issue by nobreisabelly** [cypress-io/cypress#14607]
### Current behavior
Cypress is not able to `execute` .trigger('mousedown') on an element that is listening the Hammer.js "press" event. When triggering, the handler is not executed. On the other hand, the user has no problem to interact by using the mouse click event.

### Desired behavior
The behavior should be the same as the expected when the long click event is used otherwise we cannot use Cypress in order to "simulate" the user interaction with the UI.

### Test code to reproduce
Code originally reported on: https://codepen.io/choskim/pen/RLYebL

Test code:  

```
cy.visit('./src/index.html')
cy.get('.square').trigger('mousedown')
```
Above code should be able to perform a long tap on the element, but nothing happens. 

### Versions

Cypress: 6.2.1
Hammer JS: 2.0.8

**Comment by jennifer-shehane** [comment]
What is the expected behavior for this codepen on click/press? I tried clicking and pressing on my phone outside of Cypress and nothing happens when interacting with the square. 

Can you write a failing test? 

```js
cy.visit('./src/index.html')
cy.get('.square').trigger('mousedown')

// failing assertion - whatever it should be
cy.get('.square').should('have.class', 'expand')
``

**Comment by nobreisabelly** [comment]
A long press on the square should expand it. 
Its default dimensions are 90 x 90 and after the long press it should be 225 x 225. 

Failing test case example: 
it('Should simulate the long press with left button on square and verify if size has changed to 225 x 225', () => {
        cy.get('.square').trigger('mousedown', { button: 0 })
        cy.wait(500)
        cy.get('.square').should('have.css', 'height', '90px')
        cy.get('.square').should('have.css', 'width', '225px')
    });

In this case test will fail because the height value is different then the expected. 

Just an additional information I would like to add: 
I replaced Hammer JS use for pure javascript in the codepen and I was able to perform the long press using Cypress. 
The issue in fact is using Hammer JS with Cypress.
